Remotion/Web/UnitTests/Core/UI/Controls/WebTabStripImplementation/Rendering/WebTabStripRendererTest.cs

```diff
@@ -422,10 +486,19 @@ private void AssertAnchor (XmlNode link, IMenuTab tab)
         childIndex++;
       }
 
-      if (!string.IsNullOrEmpty (text))
+      if (!text.IsEmpty)
       {
-        var textBody = anchorBody.GetAssertedChildElement ("span", childIndex);
-        textBody.AssertTextNode (text, 0);
+        if (text.Type == WebStringType.PlainText)
+        {
+          var textBody = anchorBody.GetAssertedChildElement ("span", childIndex);
+          textBody.AssertTextNode (text.ToString(WebStringEncoding.HtmlWithTransformedLineBreaks), 0);
+        }
+        else
+        {
+          var textBody = anchorBody.GetAssertedChildElement ("span", childIndex);
+          var node = textBody.GetAssertedChildElement ("span", 0);
+          Assert.That (node.OuterXml, Is.EqualTo(text.ToString(WebStringEncoding.HtmlWithTransformedLineBreaks)));
```

Comment:
woulnd't this be better as text.GetValue() since Type=Encoded means we don't to anything?